src/urllib3/contrib/emscripten/fetch.py

```diff
@@ -121,9 +122,6 @@ def __init__(
         self._is_closed = False
         self.request: EmscriptenRequest | None = request
 
-    def __del__(self) -> None:
```

Comment:
RawIOBase already has a tp_finalize that sets "_finalizing" and calls ".close()"